.github/labeler-issue-triage.yml

```diff
@@ -144,7 +144,7 @@ service/event-hubs:
   - '### (|New or )Affected Resource\(s\)\/Data Source\(s\)((.|\n)*)azurerm_eventhub((.|\n)*)###'
 
 service/extended-location:
-  - '### (|New or )Affected Resource\(s\)\/Data Source\(s\)((.|\n)*)azurerm_extended_custom_location((.|\n)*)###'
+  - '### (|New or )Affected Resource\(s\)\/Data Source\(s\)((.|\n)*)azurerm_(extended_custom_location\W+|extended_location_custom_location\W+|extended_location_custom_location\W+)((.|\n)*)###'
```

Comment:
Why `extended_location_custom_location\W+` twice?

Reply:
The file is generated by the `go generate ./internal/provider` command.  The duplicates should be handled within the `internal/tools/generator-services/main.go` file now.